module/utility/attack.mjs

```diff
@@ -1,4 +1,58 @@
 export class Attack {
+    static makeOcvModifier(ocvMod, XMLID, name) {
+        return { ocvMod, XMLID, name };
+    }
+
+    static addMultipleAttack(data) {
+        if (!data.action?.maneuver?.attackKeys?.length) {
+            return false;
+        }
+        const index = data.action.maneuver.attackKeys.length;
+        const attackKey = `attack-${index}`;
+        const itemKey = data.item.actor.items.find((item) => "STRIKE" === item.system.XMLID).id;
```

Comment:
`find` can return `undefined` which will then fail for `.id`. Why, also, are we only considering `STRIKE` as the only type of multiple attack? I guess this is back to what is a multiple attack because according to 5e it's not just STRIKE?

Reply:
It's a default 'starting' attack. everyone has a STRIKE maneuver. When you are given the Multiple Attack dropdown it's populated with the selected targets and STRIKE. The rest of the options are taken from the character's attack options, and so the STRIKE could be replaced with an Energy Blast or a Martial Throw or so forth.